addons/sourcemod/gamedata/selectivebhop.games.txt

```diff
@@ -4,7 +4,7 @@
     {
         "#supported"
         {
-            "engine"    "csgo"
```

Comment:
wait where do we even use this gamedata?

Reply:
Hold on, these gamedata seem to never be used on SelectiveBhop, maybe it was pushed when we added csgo support to multiples plugin but never pushed the csgo branch ?

Reply:
Im closing and making a new pr to remove gamedata.